### ISSUE TITLE
Code flow Azure B2C silent-renew broke in 11.1.4

### ISSUE THREAD

**Issue by sdev95** [damienbod/angular-auth-oidc-client#821]
**Describe the bug**
I had to install version 11.1.4 because of the patched bug #779, which I was really happy with. I noticed after installing this version silent-renew is not working as expected. 

In 11.1.4
It seems like after logging in, it is working as expected. When you **manually clear session storage or closing the browser and reponen** it will result in: img-2. Without a refresh being able to restore the silent renew. The solution is to manually logout, open a new browser and login again.

In 11.1.3 
After working around the error of #779, I am logged in and can open, close the browser all I want, a refresh will fix the silent renew. (Clearing session storage will break the silent renew but I am not expecting users to do this.)

**To Reproduce**
1. Use this config: 
_img-0_
![image](https://user-images.githubusercontent.com/47739530/86920585-48a85500-c12a-11ea-9223-940f261af43b.png)

2. Launch the app with LogLevel.debug, see token is validation went from this: 
_img-1_
**11.1.3**
![image](https://user-images.githubusercontent.com/47739530/86920944-b81e4480-c12a-11ea-8fd2-9ec6b412a9d9.png)
to
_img-2_
**11.1.4**
![image](https://user-images.githubusercontent.com/47739530/86921369-501c2e00-c12b-11ea-9926-454cffda84eb.png)

3. As seen in the screenshot from 11.1.4, the token is not getting renewed after closing the browser and repopening it.

**Expected behavior**
Token refresh.

I hope I gave as much info as needed, if not feel free to ask me. Thanks in advance for anyone looking at this issue. :)

**Comment by Rasetti** [comment]
@sdev95 Not exactly addressing your issue, but we have a similar scenario, including Azure B2C and code flow, and we use the refresh token instead of the iframe approach. Any reason for not implementing that instead?

**Comment by sdev95** [comment]
@Rasetti Not really actually, I came from Implicit flow with silent renew because of my clients request. But I am gonna look at the refresh token approach tomorrow! Let's hope it will fix my issue. Thank you for your reply in advance!

**Comment by sdev95** [comment]
@Rasetti Ah, the reason is that our client does not want to 'deal' with logging in and out. If they load our application the application should either send them to microsofts login page or authorize instantly. Did you implement it like this as well or are your users doing manual actions? Thank you in advance

**Comment by Rasetti** [comment]
@sdev95 I don't think that the auth flow you use affects how you implement the "auto login" feature. Regardless of the flow, you can just trigger the authorization when the app initializes, in case the user is not already authenticated.

In its simplest form, it would be something like this:

```
 oidcSecurityService.checkAuth()       
        .subscribe(isAuthenticated => {
          if (isAuthenticated) {
oidcSecurityService.authorize();
            }
        });
```

**Comment by sdev95** [comment]
@Rasetti You actually made me realize I used checkAuthIncludingServer method instead of the checkAuth method. Combined with adding offline_acces to my scope fixed version 11.1.4 with silent-renew. Now it seems to be working as exptected. 

I would like to thank you, you made my day a lot brighter :).

@damienbod Not sure if you want me to close this issue, since I worked around my initial issue. You can close it if you want.

**Comment by Rasetti** [comment]
Awesome, I'm glad it helped!
